src/components/UI/Card.vue

```diff
@@ -18,7 +18,7 @@ const classes = ref(getStyles(props, 'cardStyles'))
 
 <template>
     <div :class="classes.containerClass">
-        <div class="w-full">
+        <div class="w-100 ">
```

Comment:
I don't know if this was a merge thing, but w-full makes the element 100%. w-100 is not a valid Tailwind value.

https://tailwindcss.com/docs/width